core/arch/arm/plat-k3/main.c

```diff
@@ -49,6 +49,17 @@ void console_init(void)
 	register_serial_console(&console_data.chip);
 }
 
+static TEE_Result console_runtime(void)
+{
+	/* Console is only used for errors after this point */
+	if (IS_ENABLED(CFG_CONSOLE_RUNTIME))
```

Comment:
if this is to be generic, this function could be implemented in _core/kernel/console.c_